pycytominer/aggregate.py

```diff
@@ -27,6 +29,8 @@ def aggregate(
     Arguments:
     population_df - pandas DataFrame to group and aggregate
     strata - [default: ["Metadata_Plate", "Metadata_Well"]] list indicating the columns to groupby and aggregate
+    compute_object_count - [default: False] determine whether to compute object counts
```

Comment:
in preparation for pycytominer release, can you make sure that any comment docstring that you touch is written in numpy sphinx format? A comprehensive example is here: https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html

I've used it with some luck in [`cytominer_eval`](https://github.com/cytomining/cytominer-eval/blob/master/cytominer_eval/evaluate.py#L34)

Reply:
Does [this](https://github.com/cytomining/pycytominer/pull/129/commits/b2446a4b7b580474d8ed4efa9511643a8109a5c5) look right? If so I will change the documentation for all the functions in these 3 or 4 files that I have touched.

Reply:
looks great 👍 please use consistent punctuation though